store/store.go

```diff
@@ -765,3 +765,120 @@ func (s *SnapUbuntuStoreRepository) SuggestedCurrency() string {
 	}
 	return s.suggestedCurrency
 }
+
+// Optionally specify parameters to the store purchase.
+// Currency: ISO 4217 code as string
+// BackendID: The payment backend to use, e.g. "credit_card", "paypal".
+// MethodID: The unique identifier for the payment method, retrieved from the pay server.
+type BuyOptions struct {
+	Currency  string
```

Comment:
Can we please move the field comments into the fields themselves:

```
Currency string   // ISO 4217 code
BackendID string  // "credit_card", "paypal", etc
MethodID int64    // ... [1]
```

[1] What does that mean? We know it's a payment method id from its context already, so no need to say it again.. but what is a payment method? I know it's not paypal/credit card/etc since we have backend id.

Then, this should most likely be an int rather than an int64.